packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -325,31 +331,31 @@ export function fallbackApplePalette(): ThemeColorDefinition {
     inputTextHovered: applePlatform.textColor,
     inputPlaceholderText: applePlatform.placeholderTextColor,
 
-    // Set the fallback default button palette to communicationBlue,
-    // and use communicationBlueShade20 to emulate the NSColor pressed system effect
-    buttonBackgroundChecked: fluentUIApple.communicationBlue,
-    buttonBackgroundHovered: fluentUIApple.communicationBlue,
-    buttonBackgroundCheckedHovered: fluentUIApple.communicationBlue,
-    buttonBackgroundPressed: fluentUIApple.communicationBlueShade20,
-    buttonBackgroundDisabled: fluentUIApple.brandBackgroundDisabled,
-    buttonText: fluentUIApple.neutralInverted,
-    buttonTextHovered: fluentUIApple.neutralInverted,
-    buttonTextChecked: fluentUIApple.neutralInverted,
-    buttonTextCheckedHovered: fluentUIApple.neutralInverted,
-    buttonTextPressed: fluentUIApple.neutralInverted,
-    buttonTextDisabled: fluentUIApple.brandForegroundDisabled,
+    // Set the default button tokens to match the Acrylic Button style
```

Comment:
what's the reason to change from primary to acrylic?

Reply:
This was actually a mismatch between the JS theme and the native module. The native module had the default as acrylic, the JS had the default as primary. That was a mistake from me because I flip-flopped on that when I wrote the apple theme. I believe design wants the default to be acrylic